packages/tools/src/tools/base/ContourSegmentationBaseTool.ts

```diff
@@ -252,6 +252,11 @@ abstract class ContourSegmentationBaseTool extends ContourBaseTool {
       fillOpacity = mergedConfig.fillAlphaInactive ?? fillOpacity;
     }
 
+    // Change the line thickness when the mouse is over the contour segment
+    if (annotation.highlighted && mergedConfig.activeSegmentOutlineWidthDelta) {
```

Comment:
it is not about highlighted, it is about if it is active it should have that delta

Reply:
Fixed